src/server/webkit/wkInterceptableRequest.ts

```diff
@@ -126,6 +126,7 @@ export class WKInterceptableRequest implements network.RouteDelegate {
       requestStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.requestStart) : -1,
       responseStart: timingPayload ? wkMillisToRoundishMillis(timingPayload.responseStart) : -1,
     };
+
```

Comment:
There are a number of errant extra newlines that I'll clean up. They are an artifact of refactoring within this MR.